pyrateoptics/raytracer/helpers.py

```diff
@@ -191,17 +189,18 @@ def generate_cone_xy_bilinear(
     return pilotbundles
 
 
-def build_pilotbundle_complex(surfobj, mat, (dx, dy), (phix, phiy), Elock=None, 
+def build_pilotbundle_complex(surfobj, mat, xxx_todo_changeme5, xxx_todo_changeme6, Elock=None, 
                       kunitvector=None, lck=None, wave=standard_wavelength, 
                       num_sampling_points=3):
-    def generate_cone(direction_vec, lim_angle, (dx, dy), num_pts_dir):
+    (dx, dy) = xxx_todo_changeme5
+    (phix, phiy) = xxx_todo_changeme6
+    def generate_cone(direction_vec, lim_angle, xxx_todo_changeme2, num_pts_dir):
         """
         Generates cone for direction vector on real S_6 
         (x^2 + y^2 + z^2 + u^2 + v^2 + w^2 = 1)
         and generates cartesian raster for x and y
         """
-
-        # generate xy sampling and set pilot ray at position 0 in array
```

Comment:
This line was accidentially removed. Please reinsert it into the code. Thank you :-)

Reply:
(The comment in line 204)